src/l500/l500-motion.cpp

```diff
@@ -138,13 +118,13 @@ namespace librealsense
         hid_ep->register_option(RS2_OPTION_GLOBAL_TIME_ENABLED, enable_global_time_option);
 
         // register pre-processing
-        bool enable_imu_correction = false;
         std::shared_ptr<enable_motion_correction> mm_correct_opt = nullptr;
 
         //  Motion intrinsic calibration presents is a prerequisite for motion correction.
         try
         {
-            if (_mm_calib)
+            // L515 motion correction with IMU supported from FW version 1.4.0.10
+            if (_fw_version >= firmware_version("1.4.0.10"))
```

Comment:
I evaluated and thought about it, the mismatch actually partly because of he get_intrinsic is not implemented on L515, fixed that. on the other side, if we decide enable/disable the option in the ctor, it went back to the first PR where the table is read similay and conditioned the option creation on that. it broke the lazy mechanism.

what sergey suggested to use default intrinsic in case invalid table actually simpler for user to understand, also we avoid the options disable/enable. the option will always appear, just on/off.

in rs-enumeration-devices -c, for devices that's not calibrated or has invalid table, the default intrinsic will be used and dumped out. we claim the device is calibrated at factory, in this case, it's more consistent, an identify matrix and zero bias will be used for the imu device, i.e., no additional motion correction unless re-calibrated.